dotCMS/src/main/java/com/dotcms/rendering/velocity/services/PageContextAPI.java

```diff
@@ -80,7 +80,7 @@ public PageContextBuilder(final IHTMLPage htmlPage, final User user, final PageM
 
     }
 
-    public PageContextBuilder(final IHTMLPage htmlPage, final User user, final PageMode mode)
+    public PageContextAPI(final IHTMLPage htmlPage, final User user, final PageMode mode)
```

Comment:
I am not sure if API is the best posfix for the PageContent, since it does not follow our API approach such as  implement an interface and be provided by the APILocator.

Reply:
Agree... does not seem to be following our standard for API classes